octue/resources/analysis.py

```diff
@@ -115,16 +115,13 @@ def finalise(self, output_dir=None, save_locally=False, upload_to_cloud=False, c
         :param str bucket_name:
         :return dict: serialised strings for values and manifest data.
         """
-        serialised_strands = {}
+        serialised_strands = {"output_values": None, "output_manifest": None}
 
-        for output_strand in OUTPUT_STRANDS:
```

Comment:
I believe this means that OUTPUT_STRANDS isn't used anymore... shouldn't that be a pylint error on commit?

Reply:
It's still used but also I don't think that would be an error because it's a global variable that could be imported by other packages or users